ui/tests/integration/components/auth-jwt-test.js

```diff
@@ -51,8 +51,6 @@ const OIDC_AUTH_RESPONSE = {
   },
 };
 
-const WAIT_TIME = 50;
```

Comment:
Oh ha, that must've got left in somehow - I guess that means ESLint failure isn't failing the tests either... need to look at that.

Reply:
Oh yeah, I meant to ask about that. I was just trusting that we weren't using it like the warning said, but I can put it back if we want to figure that out separately

Reply:
Nope, it's good that it's not there! Just should make sure ESLint is working as expected in CI.